### DISCUSSION TITLE
Personal project as hub for projects in various repositories

### DISCUSSION THREAD

**Discussion by EricaThompson** [community/community#14224] [Other Feature Feedback, Questions, & Ideas]
It would be awesome if I could consolidate all of my projects in various repositories into my personal project as an overview of everything I'm working on or a selection of projects.

**Comment by GoaGill** [comment]
> It would be awesome if I could consolidate all of my projects in various repositories into my personal project as an overview of everything I'm working on or a selection of projects.

**Reply by EricaThompson** [reply]
@GoaGill I only see a quote of what I said, did you mean to add more? 🙂
